mlir/test/Dialect/SparseTensor/minipipeline_vector.mlir

```diff
@@ -1,10 +1,14 @@
 // RUN: mlir-opt %s --sparsification-and-bufferization        | FileCheck %s --check-prefix=CHECK-NOVEC
 // RUN: mlir-opt %s --sparsification-and-bufferization="vl=8" | FileCheck %s --check-prefix=CHECK-VEC
+// RUN: mlir-opt %s --sparsification-and-bufferization="parallelization-strategy=any-storage-any-loop" | FileCheck %s --check-prefix=CHECK-PARA
```

Comment:
rather than piggy backing it to the minipipelin_vector.mlir test,
can we have a new minipipeline_parallel.mlir test and have the
with and without par test there?